src/app/submission/SubmissionHandler.ts

```diff
@@ -47,10 +55,27 @@ export class SubmissionHandler {
 
     await submission.parse(message);
     await submission.save();
-    //Retrieve form definition and store in S3
 
-    //Retrieve attachments and store in S3
+    if (submission.reference) {
+      await this.sendEvent(submission.reference);
+    }
+  }
 
-    //Store submission in DynamoDB
+  async sendEvent(reference: string) {
+    const client = new EventBridgeClient();
+    await client.send(new PutEventsCommand({
+      Entries: [
+        {
+          Source: 'Submissionstorage',
+          DetailType: 'New Form Submitted',
```

Comment:
```suggestion
          DetailType: 'New Form Processed',
```
Gezien we de event pas na verwerking aftrappen?